site/layouts/index.html

```diff
@@ -14,32 +14,23 @@
 </section>
```

Comment:
We should briefly walk through this page and what you were wanting to accomplish with these changes/what order we'd want the layout ot be in.

Reply:
Okay